### PR TITLE
Spring Clean Interfaces and imports

### PR DESCRIPTION
**Pull request by selvach**
- Prefix all interfaces with an "I"
- Removes `export` where the interface is never actually used outside of that file and (usually) rename this interface to `IProps`
- Remove unused imports
- Switch to preferred function style

### CODE REVIEW

File: src/components/BrandedButton/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ActivityIndicator, StyleSheet, Text, View } from 'react-native';
 
 import { colors, fontStyles } from '@theme';
 
-import { ClickableProps } from '../Text';
+import { IClickableProps } from '../Text';
 import { ITest } from '../types';
 
 interface BrandedButtonProps extends ITest {
```
Comment (petermason) [inline src/components/BrandedButton/index.tsx:10]: IBrandedButtonProps?

File: src/components/Cards/BaseShareApp.tsx [hunk 2]
```diff
@@ -50,12 +50,7 @@ export const share = async (prefix: string) => {
   shareApp(message);
 };
 
-export const BaseShareAppCard: React.FC<BaseShareAppCardProps> = ({
-  primaryText,
-  secondaryText,
-  ctaTitle,
-  onSharePress,
-}) => {
+export const BaseShareAppCard: React.FC<IProps> = ({ primaryText, secondaryText, ctaTitle, onSharePress }) => {
```
Comment (petermason) [inline src/components/Cards/BaseShareApp.tsx:53]: function BaseShareAppCard({ primaryText, secondaryText, ctaTitle, onSharePress}:IProps) {
 ...
}
prefered style

File: src/components/Cards/ShareApp.tsx [hunk 3]
```diff
@@ -2,9 +2,9 @@ import React from 'react';
 
 import i18n from '@covid/locale/i18n';
 
-import { CommonShareProps, BaseShareAppCard, share } from './BaseShareApp';
+import { ICommonShareProps, BaseShareAppCard, share } from './BaseShareApp';
 
-export const ShareAppCard: React.FC<CommonShareProps> = (props) => {
+export const ShareAppCard: React.FC<ICommonShareProps> = (props) => {
```
Comment (petermason) [inline src/components/Cards/ShareApp.tsx:7]: use function and destructure props

File: src/components/Cards/ShareAppViral.tsx [hunk 4]
```diff
@@ -4,9 +4,9 @@ import { AreaStatsResponse } from '@covid/core/user/dto/UserAPIContracts';
 import i18n from '@covid/locale/i18n';
 import { isAndroid } from '@covid/utils/platform';
 
-import { CommonShareProps, BaseShareAppCard, shareUrl, shareApp } from './BaseShareApp';
+import { ICommonShareProps, BaseShareAppCard, shareUrl, shareApp } from './BaseShareApp';
 
-interface Props extends CommonShareProps {
+interface Props extends ICommonShareProps {
```
Comment (petermason) [inline src/components/Cards/ShareAppViral.tsx:9]: IProps ?

File: src/components/Cards/VaccineRegistryCallout.tsx [hunk 5]
```diff
@@ -5,9 +5,9 @@ import { CalloutBox } from '@covid/components/CalloutBox';
 import { colors } from '@theme';
 import appCoordinator from '@covid/features/AppCoordinator';
 
-import { CommonShareProps } from './BaseShareApp';
+import { ICommonShareProps } from './BaseShareApp';
 
-export const VaccineRegistryCallout: React.FC<CommonShareProps> = () => {
+export const VaccineRegistryCallout: React.FC<ICommonShareProps> = () => {
```
Comment (petermason) [inline src/components/Cards/VaccineRegistryCallout.tsx:10]: use function not React.FC 
doesn't have props?

**Review by petermason — COMMENTED** [review]
Stopped reviewing as a my early comments are applicable throughout

**Comment by selvach** [comment]
@petermason Ready for review :)

**Review by petermason — APPROVED** [review]

**Pull request merged**